src/atomicvar.h

```diff
@@ -5,11 +5,14 @@
  *
  * atomicIncr(var,count) -- Increment the atomic counter
  * atomicGetIncr(var,oldvalue_var,count) -- Get and increment the atomic counter
+ * atomicIncrGet(var,oldvalue_var,count) -- Increment and get the atomic counter
  * atomicDecr(var,count) -- Decrement the atomic counter
  * atomicGet(var,dstvar) -- Fetch the atomic counter value
  * atomicSet(var,value)  -- Set the atomic counter value
  * atomicGetWithSync(var,value)  -- 'atomicGet' with inter-thread synchronization
  * atomicSetWithSync(var,value)  -- 'atomicSet' with inter-thread synchronization
+ * atomicFlagTestSet(var, testres_var) -- Set on the flag and put the previous state in testres_var
```

Comment:
```suggestion
 * atomicFlagTestSet(var,testres_var) -- Set on the flag and put the previous state in testres_var
```
just to conform to the style of other lines in that block.